docs/guides/cli/20_proxy-and-tunnel.mdx

```diff
@@ -0,0 +1,376 @@
+---
+id: proxy-and-tunnel
+title: Ory Proxy and Ory Tunnel
+---
+
+The CLI contains two important tools for local development and self-hosted instances
+of the Ory Identity Service (Kratos): 
```

Comment:
It doesn't really have anything to do with self-hosted instances, but it can be used for it. On self-hosted instances you could just deploy your own Kratos locally as well for testing, so it will be a rare occurrence.

The proxy/tunnel is there to help you with your application development locally and can also be there for you in production when you do not have your domain added to Ory. The problem this is solving is to circumvent the browser CORS policy. Remember if you use a tool like `curl` you can communicate fine with the Ory APIs, but if you are doing it the browser it will complain because your current domain (e.g. localhost:4000 or www.example.com) doesn't match up with the ory domain (e.g. your-slug.projects.oryapis.com). 

So the proxy/tunnel is now the new `curl` doing the requests outside of the browser's scope and your browser is doing requests to the proxy/tunnel directly which is also "hosting" your application for you on the same domain (e.g. localhost:4000 or www.example.com)